packages/shared_preferences/shared_preferences_linux/example/integration_test/shared_preferences_test.dart

```diff
@@ -33,7 +34,7 @@ void main() {
     SharedPreferencesLinux preferences;
 
     setUp(() async {
-      preferences = SharedPreferencesLinux.instance;
+      preferences = SharedPreferencesStorePlatform.instance;
```

Comment:
Rather than add a dependency on the interface package, you could just do `preferences = SharedPreferencesLinux()`, right?

Reply:
done